packages/mobile/src/services/services.ts

```diff
@@ -53,6 +54,9 @@ const personalInfoApi = new PersonalInfoApi({
 
 const appService = new AppService();
 
+const pushNotification = new PushNotificationService();
+pushNotification.init();
```

Comment:
Only imports/exports are allowed to be here.
Place init() call into App service